EarlGrey/Matcher/GREYMatchers.m

```diff
@@ -438,9 +438,16 @@ @implementation GREYMatchers
       return NO;
     }
     NSInteger row = [element selectedRowInComponent:column];
-    NSString *rowLabel = [element.delegate pickerView:element
-                                          titleForRow:row
-                                         forComponent:column];
+    NSString *rowLabel;
+    if ([element.delegate respondsToSelector:@selector(pickerView:attributedTitleForRow:forComponent:)]) {
+        rowLabel = [element.delegate pickerView:element
```

Comment:
By fallback Hao means that you should call pickerView:titleForRow:forComponent: method and look at it's value instead.

As per documentation:
If you implement both this method (i.e. pickerView:attributedTitleForRow:forComponent:) and the 
pickerView:titleForRow:forComponent:
method, the picker view prefers the use of this method. However, if your implementation of this method returns nil, the picker view falls back to using the string returned by the pickerView:titleForRow:forComponent: method.